.shellcheckrc

```diff
@@ -0,0 +1,31 @@
+# shellcheck configuration file
+# see: https://github.com/koalaman/shellcheck/wiki
+
+# Allow following sourced files that are not specified in the command,
+# we need this because we specify one file at a time in order to trivially
+# detect which files are failing
+external-sources=true
+
+# Currently disabled these errors will take care of them later
+
+# Not following: (error message here)
+# https://github.com/koalaman/shellcheck/wiki/SC1091
+disable=SC1091
+# Since you double quoted this, it will not word split, and the loop will only run once
+# https://github.com/koalaman/shellcheck/wiki/SC2066
+disable=SC2066
+# Double quote to prevent globbing and word splitting
+# https://github.com/koalaman/shellcheck/wiki/SC2086
+disable=SC2086
+# foo appears unused. Verify it or export it
+# https://github.com/koalaman/shellcheck/wiki/SC2034
+disable=SC2034
+# This {/} is literal. Check if ; is missing or quote the expression.
+# https://github.com/koalaman/shellcheck/wiki/SC1083
+disable=SC1083
+# Declare and assign separately to avoid masking return values
+# https://github.com/koalaman/shellcheck/wiki/SC2155
+disable=SC2155
+# Quote this to prevent word splitting
+# https://github.com/koalaman/shellcheck/wiki/SC2046
+disable=SC2046
```

Comment:
It seems that some settings of violation ignoring are not copied from `verify-shellcheck.sh` here, such as `SC2002`.

Is there any reason?

Reply:
These rules are redundant. As stated in `verify-shellcheck.sh` "Currently disabled these errors will take care of them later". However, they have already been fixed and shouldn't be ignored.

Reply:
I see. Thank you for the clarifications.